### PR TITLE
Mermaid markdown preview and syntax highlighting

### PR DESCRIPTION
**Pull request by vitaliy-guliy**
Signed-off-by: Vitaliy Gulyy <vgulyy@redhat.com>

Publish .vsix binaries for

**Markdown Preview Mermaid Support**
- https://marketplace.visualstudio.com/items?itemName=bierner.markdown-mermaid

Repository
- https://github.com/mjbvz/vscode-markdown-mermaid

**Mermaid Markdown Syntax Highlighting**
- https://marketplace.visualstudio.com/items?itemName=bpruitt-goddard.mermaid-markdown-syntax-highlighting

Repository
- https://github.com/bpruitt-goddard/vscode-mermaid-syntax-highlight

### CODE REVIEW

File: vscode-mermaid-syntax-highlight/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
```
Comment (benoitf) [inline vscode-mermaid-syntax-highlight/Dockerfile:2]:
```suggestion
# Copyright (c) 2021 Red Hat, Inc.
```

File: vscode-mermaid-syntax-highlight/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+FROM registry.access.redhat.com/ubi8/nodejs-12
```
Comment (benoitf) [inline vscode-mermaid-syntax-highlight/Dockerfile:10]: you should add the specific tag of nodejs-12 image

also comment one line before the link to know the tag like:
```
# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8/nodejs-12
FROM registry.access.redhat.com/ubi8/nodejs-12:1-64 as runtime
```

Comment (vitaliy-guliy) [reply vscode-mermaid-syntax-highlight/Dockerfile:10]: Why not latest tag as for other dockerfiles?

File: vscode-mermaid-syntax-highlight/Dockerfile (hunk 1 above)
Comment (vitaliy-guliy) [reply vscode-mermaid-syntax-highlight/Dockerfile]: Done

File: vscode-mermaid-syntax-highlight/Dockerfile (hunk 2 above)
Comment (benoitf) [reply vscode-mermaid-syntax-highlight/Dockerfile:10]: it's probably a mistake elsewhere, we should always stick to given tags to reproduce the build

Comment (vitaliy-guliy) [reply vscode-mermaid-syntax-highlight/Dockerfile]: OK, then I will add tags for all nodejs-12 images

File: Dockerfile [hunk 3]
```diff
@@ -7,7 +7,8 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-FROM registry.access.redhat.com/ubi8/nodejs-12
+# https://catalog.redhat.com/software/containers/ubi8/nodejs-12/5d3fff015a13461f5fb8635a?tag=1-64
```
Comment (benoitf) [inline Dockerfile:10]: you should bump to 1-70 (latest avalailable now without security issues)
https://catalog.redhat.com/software/containers/ubi8/nodejs-12/5d3fff015a13461f5fb8635a?tag=1-70

Comment (vitaliy-guliy) [reply Dockerfile]: Done

**Review by benoitf — APPROVED** [review]
👍

**Comment by benoitf** [comment]
@ericwill could you please add https://github.com/orgs/redhat-developer/teams/che team as committers to this repo
seems you're the only one to merge PR there ?

**Comment by ericwill** [comment]
> @ericwill could you please add https://github.com/orgs/redhat-developer/teams/che team as committers to this repo
> seems you're the only one to merge PR there ?

Done

**Pull request merged**